packages/ripple/src/runtime/internal/client/portal.js

```diff
@@ -28,16 +32,26 @@ export function Portal(_, props) {
       destroy_block(b);
     }
 
+    domStart = domEnd = null;
+
     anchor = create_text();
     /** @type {Element} */ (target).append(anchor);
 
     const cleanup_events = handle_root_events(/** @type {Element} */ (target));
 
     b = branch(() => /** @type {(anchor: Node) => void} */ (children)(/** @type {Text} */ (anchor)));
 
+    if (b && b.s) {
+      domStart = b.s.start;
+      domEnd = b.s.end;
+    }
+
     return () => {
       cleanup_events();
       /** @type {Text} */ (anchor).remove();
+      if(domStart !== null && domEnd !== null) {
```

Comment:
seems like these can also be undefined.


```suggestion
      if(domStart != null && domEnd != null) {
```